source/operations/install-deploy-manage/expand-minio-deployment.rst

```diff
@@ -28,6 +28,24 @@ The procedure on this page expands an existing
 :ref:`distributed <deploy-minio-distributed>` MinIO deployment with an
 additional server pool. 
 
+Rebalancing Pools
+-----------------
+
+MinIO optimizes storage of objects across available pools by writing new objects to the server pool with the most free space compared total amount of free space on all available server pools.
+For the calculations and more details, see :ref:`Writing Files <minio-writing-files>` below.
+As a result, MinIO does not perform the costly action of rebalancing objects from older pools to newer pools.
+Based on write preference calculation, new writes default to the new pool as it has the most free space compared to the other pools until the new pool reaches a similar level of contents compared to the other pools.
```

Comment:
Suggest the following order here:

Line 34
Line 36 (drop "As a result")
Line 37 (Start with "Instead, new writes typically...")
Line 35 ("For more information on write preference calculation logic, see X")

Reply:
That makes so much more sense. Done.